pkg/util/util.go

```diff
@@ -84,6 +84,17 @@ func FileExists(filename string) (bool, error) {
 	return true, nil
 }
 
+// Verify block device is not in use by calling openFile with O_EXCL flag.
+// Returns nil if device not in use, but a non-nil return value means
+// that either the device is in use or another error occurred during OpenFile.
+func CheckDeviceInUseBySystem(pathname string) error {
+	fp, err := os.OpenFile(pathname, os.O_RDONLY|os.O_EXCL, 0700)
+	if fp != nil {
+		fp.Close()
```

Comment:
It's more customary to check err != nil. unless you've read the function documentation (or implementation) it's not wise to touch the non-error return components without checking the error.